.gitignore

```diff
@@ -15,3 +15,8 @@ result-*
 /pkgs/development/libraries/qt-5/*/tmp/
 /pkgs/desktops/kde-5/*/tmp/
 /pkgs/development/mobile/androidenv/xml/*
+.envrc
+.direnv
+shell.nix
+.history
+pkgs/applications/search/deezer-downloader/
```

Comment:
I don't have anything against expanding gitignore, but another-package-specific entry is probably too unrelted